src/encoded/schemas/file_other.json

```diff
@@ -0,0 +1,41 @@
+{
+    "title": "Other file",
+    "description": "Other files to ingest.",
+    "id": "/profiles/file_other.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["file_format", "award", "lab"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
```

Comment:
@aschroed and @willronchetti, is accession a general-purpose term or is it only relevant to specific tasks? Should we be including it here?